.github/ISSUE_TEMPLATE/request.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 💡 Feature Request
+about: I have a suggestion (and might want to implement myself 🙂)!
+---
+
+<!--
+
+Vote on feature requests by adding a 👍. This helps maintainers prioritize what
```

Comment:
This sentence seems to indicate that the maintainers decide what they work on. What do you think about phrasing it a little more open?

> This helps us to prioritise the important issues

> This helps the community to prioritise what to start with